extensions/amp-fx-flying-carpet/0.1/test/test-amp-fx-flying-carpet.js

```diff
@@ -44,7 +44,11 @@ describe('amp-fx-flying-carpet', () => {
       iframe = i;
       toggleExperiment(iframe.win, 'amp-fx-flying-carpet', true);
 
-      iframe.doc.body.style.height = '400vh';
+      const bodyResizer = iframe.doc.createElement('div');
```

Comment:
different way of sizing the body's height.  We now install the runtime Css when creating the test iframe and therefore iframe.doc.body.style.height = '400vh' was not being applied because of !important! rules coming from runtime Css was overriding it.